ingestion/src/metadata/ingestion/connections/test_connections.py

```diff
@@ -329,6 +329,8 @@ def test_connection_db_common(
         timeout_seconds=timeout_seconds,
     )
 
+    kill_active_connections(engine)
```

Comment:
also this works for the db common, but should we call this in other services as well?

Reply:
added for multi db as well